config.md

```diff
@@ -119,11 +119,11 @@ For Windows, see [mountvol][mountvol] and [SetVolumeMountPoint][set-volume-mount
 
 ## <a name="configProcess" />Process
 
-**`process`** (object, OPTIONAL) specifies the container process.
+**`process`** (object, OPTIONAL) specifies a process to run inside the container.
   This property is REQUIRED when [`start`](runtime.md#start) is called.
 
-* **`terminal`** (bool, OPTIONAL) specifies whether a terminal is attached to that process, defaults to false.
-  As an example, if set to true on Linux a pseudoterminal pair is allocated for the container process and the pseudoterminal slave is duplicated on the container process's [standard streams][stdin.3].
+* **`terminal`** (bool, OPTIONAL) specifies whether a terminal is attached to the process, defaults to false.
+  As an example, if set to true on Linux a pseudoterminal pair is allocated for the process and the pseudoterminal slave is duplicated on the process's [standard streams][stdin.3].
```

Comment:
Let's punt that to a new PR. There are too many lines starting with two-space indents